Solutions/STM32F4DISCOVERY/scatterfile_gcc_missing_symbols.xml

```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <ScatterFile xmlns="http://schemas.microsoft.com/netmf/ScatterfileSchema.xsd">
+  <!-- 
+    - using newlib-nano adds libnosys.a, which calls sbrk, which requires the 'end' symbol
+    - this is pulled in by a _printf imported in function _Crypto_StepRSAOperation @ Crypto.lib(rsa_zenith.obj) 
+  -->
+  <Provide Name="end" />

```

Comment:
Shouldn't be the 'end' symbol added to the linker script (scatterfile) instead?

It needs to be placed at the lowest allocatable memory address not occupied by the program or its data.